packages/client/tests/functional/issues/20261-group-by-shortcut/tests.ts

```diff
@@ -0,0 +1,55 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(() => {
+  beforeAll(async () => {
+    await prisma.round.create({ data: { teamName: 'Red', points: 5 } })
+    await prisma.round.create({ data: { teamName: 'Blue', points: 7 } })
+    await prisma.round.create({ data: { teamName: 'Red', points: 4 } })
+    await prisma.round.create({ data: { teamName: 'Blue', points: 3 } })
+  })
+
+  test('works with a scalar in "by"', async () => {
+    const result = await prisma.round.groupBy({
+      _sum: {
+        points: true,
+      },
+      by: 'teamName',
+      orderBy: { teamName: 'asc' },
+    })
+
+    expect(result).toEqual([
+      {
+        _sum: {
+          points: 10,
+        },
+        teamName: 'Blue',
+      },
+      {
+        _sum: {
+          points: 9,
+        },
+        teamName: 'Red',
+      },
+    ])
+  })
+
+  test('works with a scalar in "by" and no other selection', async () => {
+    const result = await prisma.round.groupBy({
+      by: 'teamName',
+      orderBy: { teamName: 'asc' },
+    })
```

Comment:
Should we also add type-level tests? Not sure if we needed to update the underlying implementation?

Reply:
For Client extensions, groupBy types are "disconnected" from the centralized types we introduced via extensions in `GetResult.ts`, I think. So we may need to add tests for client extensions too (see [code](https://github.com/prisma/prisma/blob/62e09f34b9b0215817d085907d349eeaa842cc35/packages/client/src/runtime/core/types/GetResult.ts/#L127-L129)).

Reply:
Fixed extensions as well.

Reply:
Nice, thank you.